### PR TITLE
split landing into two different containers

### PR DESCRIPTION
**Pull request by erizocosmico**
Needs #73

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,7 @@ PROJECT = landing
 COMMANDS = api
 CODECOV_TOKEN ?=
 DESTINATION ?= public
+DOCKERFILES = Dockerfile:landing Dockerfile.api:landing-api
```
Comment (rporres) [inline Makefile]: This is out of the scope of this PR, but I assume this variable should contain Dockerfile.slackin at some point in the future...

File: Dockerfile.api [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.6
+
+COPY bin /bin/
+
+CMD ["/bin/api"]
```
Comment (rporres) [inline Dockerfile.api]: Can this work without configuration?

**Comment by Serabe** [comment]
I trust @erizocosmico and @rporres for this task, whenever they say it is ok, it can be merged.

File: Makefile (hunk 1 above)
Comment (erizocosmico) [inline Makefile]: Yep. Right now the makefile does not even build the slackin, we might have to do that too. I will do it in another PR.

File: Dockerfile.api (hunk 2 above)
Comment (erizocosmico) [inline Dockerfile.api]: Yep, everything is an env var now

**Comment by erizocosmico** [comment]
Do not merge before #73 is merged.

**Comment by erizocosmico** [comment]
I need to change this to remove the proxying and add the slackin image and blah blah blah blah

**Comment by erizocosmico** [comment]
@rporres but it stills needs the proxies removed...

El jue., 20 jul. 2017 a las 11:21, Rafael Porres Molina (<notifications@github.com<mailto:notifications@github.com>>) escribió:

@rporres approved this pull request.

🚢 it!

**Comment by rporres** [comment]
Yes, it needs to have the proxies removed.

**Comment by erizocosmico** [comment]
Proxies have been removed

**Comment by erizocosmico** [comment]
Slackin build added too

**Comment by dpordomingo** [comment]
so... in which state is this one? is it now ready to be merged @rporres?

**Comment by rporres** [comment]
I see it ok to be merged. The only problem would be if you have to do a hotfix in current prod, which is not prepared to work without the proxy. I guess you would have to branch and fix it from there.

**Comment by dpordomingo** [comment]
If we have to do a hotfix in current landing, I think we should do it in the old repo (the private one, because we changed many other things....)
I think the Landing will be deployed only in the new infrastructure, but @Serabe would confirm it.

**Comment by rporres** [comment]
ok. I was not aware of the repo changes and stuff... It should be fine, then

**Comment by Serabe** [comment]
Current state is not deployable in a hotfix. The process is what David said plus porting the fix to this repo if needed.

Merging this, thank you all!

**Pull request merged**